### PR TITLE
Updated Desktop URLs and removed CSP and EE URLs

### PR DESCRIPTION
**Pull request by usha-mandya**
- Removed Get Docker drop-down from top navigation
- Added a link to Get started docs
- Fixed the URL to Docker Products

### CODE REVIEW

**Comment by usha-mandya** [comment]
@thaJeztah Could you PTAL?

File: _includes/global-header.html [hunk 1]
```diff
@@ -10,24 +10,18 @@
                     <div class="navbar-collapse collapse">
                         <ul class="primary nav navbar-nav">
                             <li><a href="https://docker.com/why-docker">Why Docker?</a></li>
-                            <li><a href="https://docker.com/get-started">Product</a></li>
+                            <li><a href="https://www.docker.com/products">Products</a></li>
```
Comment (thaJeztah) [inline _includes/global-header.html]: Looks like we use `www.` as the canonical URL, so we should probably do a find/replace across the repository

`https://docker.com/` -> `https://www.docker.com/`

File: _includes/global-header.html [hunk 2]
```diff
@@ -10,24 +10,18 @@
...
-                                    <li><a href="https://docker.com/products/docker-desktop">Mac & Windows</a></li>
-                                    <h6 class="dropdown-header">For Cloud Providers</h6>
-                                    <li><a href="https://docker.com/partners/aws">AWS</a></li>
-                                    <li><a href="https://docker.com/partners/microsoft">Azure</a></li>
+                                    <li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-mac">Mac</a></li>
+									<li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-windows">Windows</a></li>
                                     <h6 class="dropdown-header">For Servers</h6>
-                                    <li><a href="https://hub.docker.com/editions/enterprise/docker-ee-server-windows">Windows Server</a></li>
-                                    <li><a href="https://hub.docker.com/editions/enterprise/docker-ee-server-centos">CentOS</a></li>
+                                    <li><a href="https://hub.docker.com/editions/community/docker-ce-server-centos">CentOS</a></li>
```
Comment (thaJeztah) [inline _includes/global-header.html]: I'm wondering what the goal is of these links. Looking at this one (for example) will link to Docker Hub, which only has a description (and the description is actually outdated), and to _install_, it gives a link back to the documentation (https://docs.docker.com/engine/installation/linux/centos/)

File: _includes/global-header.html [hunk 3]
```diff
@@ -10,24 +10,18 @@
...
                             <li class="dropdown">
                                 <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Get Docker <span class="caret"></span></a>
                                 <ul class="dropdown-menu nav-main">
                                     <h6 class="dropdown-header">For Desktop</h6>
-                                    <li><a href="https://docker.com/products/docker-desktop">Mac & Windows</a></li>
-                                    <h6 class="dropdown-header">For Cloud Providers</h6>
-                                    <li><a href="https://docker.com/partners/aws">AWS</a></li>
-                                    <li><a href="https://docker.com/partners/microsoft">Azure</a></li>
+                                    <li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-mac">Mac</a></li>
+									<li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-windows">Windows</a></li>
```
Comment (thaJeztah) [inline _includes/global-header.html]: Looks like indentation went a bit off here

File: _includes/global-header.html (hunk 1 above)
Comment (usha-mandya) [reply _includes/global-header.html]: Sure, will fix that.

**Comment by usha-mandya** [comment]
Discussed offline with @thaJeztah. 

Looks like we surface the Get Docker URL in multiple places on the home page. Removed this section and added link to the Get started docs instead.

File: _includes/global-header.html (hunk 3 above)
Comment (usha-mandya) [reply _includes/global-header.html]: Fixed.

**Pull request merged**